source/form.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+  <body class="page__body">
+    <header class="page-header">
+      <div class="page-header__wrapper">
+        <a class="logo page-header__logo" href="index.html">
+          <img src="img/logo-icon-mobile.svg" width="33" height="38" alt="Иконка логотипа cat energy">
+          <img src="img/logo.svg" width="101" height="18" alt="Логотип cat energy">
+        </a>
+      </div>
+      <button class="page-header__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a class="main-nav__link" href="index.html">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a class="main-nav__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="main-nav__item">
+            <a class="main-nav__link main-nav__link--current" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="inner-main">
+      <section class="form">
+        <h2 class="form__title">Подбор программы</h2>
+        <p class="form__text">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+        <form action="https://echo.htmlacademy.ru" method="post" target="_blank">
+          <fieldset class="form__group">
+            <legend class="visually-hidden">Информация о коте</legend>
+            <ul class="form__list">
+              <li class="form__item">
+                <label class="form__label" for="cat-name">Имя:<sub>*</sub></label>
```

Comment:
повторяю замечание, sub лишний
```suggestion
                <label class="form__label" for="cat-name">Имя:*</label>
```